README.md

```diff
@@ -1,2 +1,180 @@
-# To-Do-list
-A **To Do List** **SPA** using **webpack**
+<a name="readme-top"></a>
+
+<div align="center">
+  <img src="Astroboy-removebg.png" alt="logo" width="120"  height="auto" />
+  <br/>
+
+  <h3><b>Hello Microverse</b></h3>
```

Comment:
- [OPTIONAL] Always ensure your README file reflects what is in your project. The title of this milestone is `Todo List`. I wonder why your README title carries `Hello Microverse`. Always try to pay attention to details. This is very important as a developer. Kindly make your title reflect your project title.